app/subscriber/src/features/my-reports/edit/content/ContentEditForm.tsx

```diff
@@ -215,7 +215,14 @@ export const ContentEditForm = React.forwardRef<HTMLDivElement | null, IContentE
             </Col>
             <Col className="sentiment">
               <Sentiment
-                value={form.content.tonePools.length ? form.content.tonePools[0].value : undefined}
+                value={
+                  form.content.versions?.[userId]?.tone !== undefined &&
```

Comment:
As far as I can tell `tone` should never be `null`

Reply:
First ime the `tone` can be null when the user did not save her/his tone and until `tone` is saved by the user, then we have a value.